pkg/tests/apis/openapi_snapshots/provisioning.grafana.app-v0alpha1.json

```diff
@@ -2445,10 +2445,67 @@
         }
       ]
     },
+    "/apis/provisioning.grafana.app/v0alpha1/namespaces/{namespace}/settings": {
+      "get": {
+        "tags": [
+          "Provisioning"
+        ],
+        "description": "Get the frontend settings for this namespace",
```

Comment:
The whole concept about "frontend settings" sounds counterintuitive to me for the APIs we are trying to build. 

It sounds like a backend for frontend not a backend services platform.

Reply:
Any better naming suggestions?  it is essentially a "view" or a "summary" of the same files -- with more details trimmed out